awx/api/serializers.py

```diff
@@ -3270,15 +3319,8 @@ class Meta:
             'verbosity': {'write_only': True,}
         }
 
-    # TODO: remove in 3.3
-    def get_fields(self):
```

Comment:
This key is totally gone from the launch endpoint now.  See: https://github.com/ansible/awx/pull/595/files#r149437139